src/App.svelte

```diff
@@ -4,19 +4,28 @@
   let yearlyIncome = 0;
   let year = '2022';
   let marrStatus = 'single';
+  let capGainsMarrStatus = 'LT_capital_gains_' + marrStatus;
   let taxes = 0;
   let afterTax = 0;
+  let capitalGainsShortTerm = 0;
+  let capitalGainsLongTerm = 0;
+  //let hasCapital = false;
 
   // @ts-ignore
   function calcTaxes() {
-      let intIncome = yearlyIncome;
+      let shortTermCapGains = capitalGainsShortTerm;
+      let longTermGapGains = capitalGainsLongTerm;
+      let intIncome = yearlyIncome + capitalGainsShortTerm; //short-term cap gains tax is considered as part of your yearly income
+      
       let curr_taxes = 0;
 
       // grab correct tax data for year
       // @ts-ignore
       let currMap = tax_data.yearToTax[year];
       let rates = currMap["percents"];
+      let capGainsRates = currMap["LT_capital_gains_percents"]
       let salaries = currMap[marrStatus];
+      let capGains = currMap[capGainsMarrStatus];
```

Comment:
Is it possible to set capital gains taxes similarly to how standard deduction rates are set? I feel that implementation in prod looks a bit better.

Can you make this code look a bit more like [this](https://github.com/Larry-Gan/PCSTaxToolV2/blob/main/src/App.svelte#L12) and [this](https://github.com/Larry-Gan/PCSTaxToolV2/blob/main/src/lib/data.js#L58)?